src/mono/wasm/build/WasmApp.targets

```diff
@@ -134,7 +134,33 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_WasmGenerateAppBundle" Condition="'$(WasmGenerateAppBundle)' == 'true'">
+  <Target Name="_WasmGenerateRuntimeConfig" Condition="'$(OutputType)' != 'library'">
```

Comment:
Where did it hit the `library` case?

Reply:
And why was `'$(WasmGenerateAppBundle)' == 'true'` removed?

Reply:
> Where did it hit the `library` case?

https://github.com/dotnet/runtime/blob/main/src/tests/FunctionalTests/WebAssembly/Browser/HotReload/ApplyUpdateReferencedAssembly/ApplyUpdateReferencedAssembly.csproj#L5

Reply:
> And why was `'$(WasmGenerateAppBundle)' == 'true'` removed?

That has always been there for another target `_WasmGenerateAppBundle`, not the one I added. The diff is misleading there.

Reply:
oops, I misread that!